cli/version/version_match_test.go

```diff
@@ -0,0 +1,235 @@
+package version
```

Comment:
I understand that the rules are broken in the `tt` from time to time, but it is better to test only the public API. At least for a new code.

```suggestion
package version_test
```